File: silly.py
# https://codehs.com/sandbox/id/python-3-FJ9Q5s?filename=main.py

import asyncio
from pyppeteer import launch
import time

async def main():
    browser = await launch(options={'args': ['--no-sandbox'], 'devtools':True, 'headless':True})
    page = await browser.newPage()
    await page.goto('https://codehs.com/sandbox/id/python-3-jLlrsf')
    await page.screenshot({'path': './code.png'})
    time.sleep(2)
    # await page.screenshot({'path': './code.png'})
    for x in range(3000):
        await asyncio.sleep(0.5)
        await page.screenshot({'path': "r1.png"})
        await asyncio.sleep(10)
        await page.keyboard.press('KeyH')
        await asyncio.sleep(10)
        await page.keyboard.press('KeyH')
        await asyncio.sleep(10)
        await page.keyboard.press('KeyH')
        await asyncio.sleep(70)
        await page.mouse.click(580,125,{'button': 'left'})
        await page.screenshot({'path': "res.png"})

    await browser.close()
asyncio.get_event_loop().run_until_complete(main())

File: silly.js
const userAgent = require('user-agents');
const puppeteer = require('puppeteer');

(async () => {
  const browser = await puppeteer.launch({
    headless: true,
    args: ['--no-sandbox']
 });

  const page = await browser.newPage();
  await page.setUserAgent(userAgent.toString())
  await page.goto('https://codehs.com/sandbox/id/python-3-FuF5AE')
  await page.screenshot({'path': './codejs.png'})
  await page.waitForTimeout(2000)
//   # await page.screenshot({'path': './code.png'})
 for(x=0;x<10000;x++) {
    await page.waitForTimeout(500)
    // await page.screenshot({'path': "r1.png"})
    await page.waitForTimeout(10000)
    await page.keyboard.press('KeyH')
    await page.waitForTimeout(10000)
    await page.keyboard.press('KeyH')
//     await page.screenshot({'path': "r1.png"})
    await page.waitForTimeout(10000)
    await page.keyboard.press('KeyH')
    await page.waitForTimeout(10000)
//     await page.mouse.click(580,125,{'button': 'left'})
    await page.screenshot({'path': "resjs.png"})
 }
  await browser.close();
})();
